src/cljs/ataru/virkailija/application/view.cljs

```diff
@@ -1119,20 +1168,20 @@
               [:i.zmdi.zmdi-check-circle.zmdi-hc-lg.application-handling__eligibility-automatically-checked
                {:title (get-virkailija-translation :eligibility-set-automatically)}])]
            (if @list-opened
-             [:div.application-handling__review-state-list-opened-anchor
-              (into [:div.application-handling__review-state-list-opened
+             (into [:div.application-handling__review-state-list.application-handling__review-state-list--opened
                      {:on-click list-click}]
-                    (opened-review-state-list kw review-state-for-current-hakukohde states @lang))]
-             [:div.application-handling__review-state-selected-container
+                    (opened-review-state-list kw review-state-for-current states @lang multiple-values?))
+             [:div.application-handling__review-state-list
               [review-state-selected-row
                list-click
                (application-states/get-review-state-label-by-name
                 states
-                (or @review-state-for-current-hakukohde (ffirst states))
-                @lang)]
+                (or review-state-for-current (ffirst states))
+                @lang)
+               multiple-values?]
               (when (and (= :eligibility-state kw)
-                         (= "uneligible" @review-state-for-current-hakukohde))
-                [review-state-comment kw (keyword @current-hakukohde)])])])))))
+                         (= "uneligible" review-state-for-current))
+                (map #([review-state-comment kw %]) @current-hakukohteet))])])))))
```

Comment:
Tässä varmaan pitäis välittää `@current-hakukohteet` sellasenaan `review-state-comment` komponentille, ja muuttaa sitä niin, että se piirtää vaan yhden tekstiboxin mutta luo samalla viestin kaikille valituille hakukohteille.

Reply:
Tässä oli lisähaasteena, että monen hakukohteen valinnan syötekenttien tilan ylläpito ei ole ihan suoraviivaista. Korjasin alustavasti niin, että monelle valinnalle tilaa pidetään erikseen. Ei ihan aukoton korjaus, mutta esim. uusimman yhdistelmätilan näyttäminen aikaleimaa ylläpitämällä, johtaa haastavaan komponentinpäivitysongelmaan, kun aika leimat vaihtuu useassa komponentissa jokaisella syötteen muutoksella.